### ISSUE TITLE
flexor carpi radialis tendonitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10757]
**Preferred term label:**
flexor carpi radialis tendonitis

**Synonyms**
FCR inflammation

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Flexor carpi radialis (FCR) tendonitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001423
- label: Flexor carpi radialis tendonitis
- parents: [Tendonitis (HP:0025230)](https://hpo.jax.org/app/browse/term/HP:0025230)
- pmid: PMID:39166186
- def: Inflammation of the flexor carpi radialis (FCR) tendon, which traverses the carpus through a narrow fibro-osseous tunnel.
- comment: Flexor carpi radialis (FCR) tendinitis is an uncommon but important cause of volar radial wrist pain. The FCR tendon may become inflamed primarily or as a sequelae of carpometacarpal arthroplasty, arthritis, or trauma.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10757
- synonyms: FCR inflammation(NONE)
